aten/src/ATen/native/native_functions.yaml

```diff
@@ -6472,6 +6472,31 @@
     SparseCPU, SparseCUDA: norm_sparse
   autogen: native_norm.ScalarOpt_dim_dtype_out
 
+- func: batch_norm_with_update(Tensor input, Tensor? weight, Tensor? bias, Tensor(a!) running_mean, Tensor(b!) running_var, float momentum, float eps, bool cudnn_enabled) -> (Tensor, Tensor, Tensor, Tensor)
```

Comment:
Maybe add a comment right of this line describing what the four different tensor returns are (even I can't remember off the top of my head after staring at batchnorm for 2 years 😛 )